book/history.md

```diff
@@ -67,21 +68,23 @@ known, but by making previously-unknown connections *between known things*:
 By "association", Bush meant a trail of thought leading from one record
 to the next via a human-curated link. He imagined not just a universal
 library, but a universal way to record the results of what we learn. That is
-what today's web does.
+what today's web can do also.
 
 The web emerges
 ===============
 
-The concept of interlinked [hypertext](hypertext) documents and the
-hyperlink concept was invented in
-[1964-65](https://en.wikipedia.org/wiki/Hyperlink#History) by [Project
-Xanadu](xanadu), led by Ted Nelson.[^literary-criticism] Hypertext is text that
-is marked up with hyperlinks to other text. Sounds familiar? A web page is
-hypertext, and links between web pages are hyperlinks. The format for writing
-web pages is HTML, which is short for HyperText Markup Language. The protocol
-for loading web pages is HTTP, which is short for HyperText Transport Protocol.
-A successor system called [HES](hes) was the first to introduce the back button,
-which all browsers now have.
+The concept of interlinked [hypertext](hypertext) documents and the [hyperlink]
+concept was invented in 1964-65 by [Project Xanadu](xanadu), led by Ted
```

Comment:
```suggestion
concept was invented in 1964-65 by [Project Xanadu][xanadu], led by Ted
```

Parentheses mean an explicit URL, square brackets are a reference to a URL given elsewhere.